src/containers/checkout/checkout.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import OrderForm from './order-form';
+import Cart from '../../pages/cart';
+
+export const Checkout = () => (
+  <div>
```

Comment:
If you just need a wrapper you could use fragment `<> </>` instead of `<div>`

Reply:
fixed